<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-03-28T01:48:49 IST" name="Suite" finished-at="2024-03-28T01:49:02 IST" duration-ms="13255">
    <groups>
      <group name="ErrorValidations">
        <method signature="LoginErrorValidationTest.loginErrorValidation()[pri:0, instance:abhishektestnet.seleniumFrameworkDesign.tests.LoginErrorValidationTest@33065d67]" name="loginErrorValidation" class="abhishektestnet.seleniumFrameworkDesign.tests.LoginErrorValidationTest"/>
      </group> <!-- ErrorValidations -->
      <group name="Regression">
        <method signature="SubmitOrderTest.submitOrderTest(java.util.HashMap)[pri:0, instance:abhishektestnet.seleniumFrameworkDesign.tests.SubmitOrderTest@2e6a8155]" name="submitOrderTest" class="abhishektestnet.seleniumFrameworkDesign.tests.SubmitOrderTest"/>
      </group> <!-- Regression -->
    </groups>
    <test started-at="2024-03-28T01:48:49 IST" name="LoginErrorValidationTest" finished-at="2024-03-28T01:48:54 IST" duration-ms="4934">
      <class name="abhishektestnet.seleniumFrameworkDesign.tests.LoginErrorValidationTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:abhishektestnet.seleniumFrameworkDesign.tests.LoginErrorValidationTest@33065d67]" started-at="2024-03-28T01:48:49 IST" name="launchApplication" finished-at="2024-03-28T01:48:51 IST" duration-ms="2184" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="loginErrorValidation()[pri:0, instance:abhishektestnet.seleniumFrameworkDesign.tests.LoginErrorValidationTest@33065d67]" started-at="2024-03-28T01:48:51 IST" name="loginErrorValidation" finished-at="2024-03-28T01:48:53 IST" duration-ms="1979" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at abhishektestnet.seleniumFrameworkDesign.tests.LoginErrorValidationTest.loginErrorValidation(LoginErrorValidationTest.java:22)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:abhishektestnet.seleniumFrameworkDesign.tests.LoginErrorValidationTest@33065d67]" started-at="2024-03-28T01:48:53 IST" name="tearDown" finished-at="2024-03-28T01:48:54 IST" duration-ms="119" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- abhishektestnet.seleniumFrameworkDesign.tests.LoginErrorValidationTest -->
    </test> <!-- LoginErrorValidationTest -->
    <test started-at="2024-03-28T01:48:49 IST" name="SubmitOrderTest" finished-at="2024-03-28T01:49:02 IST" duration-ms="13255">
      <class name="abhishektestnet.seleniumFrameworkDesign.tests.SubmitOrderTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:abhishektestnet.seleniumFrameworkDesign.tests.SubmitOrderTest@2e6a8155]" started-at="2024-03-28T01:48:49 IST" name="launchApplication" finished-at="2024-03-28T01:48:51 IST" duration-ms="2070" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrderTest(java.util.HashMap)[pri:0, instance:abhishektestnet.seleniumFrameworkDesign.tests.SubmitOrderTest@2e6a8155]" started-at="2024-03-28T01:48:51 IST" name="submitOrderTest" data-provider="getDataForSubmitOrderTest" finished-at="2024-03-28T01:48:56 IST" duration-ms="4881" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Strong@00, product=ADIDAS ORIGINAL, username=abhishek@email.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrderTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:abhishektestnet.seleniumFrameworkDesign.tests.SubmitOrderTest@2e6a8155]" started-at="2024-03-28T01:48:56 IST" name="tearDown" finished-at="2024-03-28T01:48:56 IST" duration-ms="70" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:abhishektestnet.seleniumFrameworkDesign.tests.SubmitOrderTest@2e6a8155]" started-at="2024-03-28T01:48:56 IST" name="launchApplication" finished-at="2024-03-28T01:48:57 IST" duration-ms="1169" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrderTest(java.util.HashMap)[pri:0, instance:abhishektestnet.seleniumFrameworkDesign.tests.SubmitOrderTest@2e6a8155]" started-at="2024-03-28T01:48:57 IST" name="submitOrderTest" data-provider="getDataForSubmitOrderTest" finished-at="2024-03-28T01:49:02 IST" duration-ms="4803" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Strong@00, product=IPHONE 13 PRO, username=vishal@email.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrderTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:abhishektestnet.seleniumFrameworkDesign.tests.SubmitOrderTest@2e6a8155]" started-at="2024-03-28T01:49:02 IST" name="tearDown" finished-at="2024-03-28T01:49:02 IST" duration-ms="130" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- abhishektestnet.seleniumFrameworkDesign.tests.SubmitOrderTest -->
    </test> <!-- SubmitOrderTest -->
  </suite> <!-- Suite -->
</testng-results>
